softgroup/basis/common-router/src/main/java/com/softgroup/common/router/impl/MainRouter.java

```diff
@@ -0,0 +1,19 @@
+package com.softgroup.common.router.impl;
+
+import com.softgroup.common.router.api.AbstractRouterHandler;
+import com.softgroup.common.router.api.CommonRouterHandler;
+import com.softgroup.common.router.api.RouterHandler;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by yuriy on 28.02.17.
+ */
+@Component
+public class MainRouter
+        extends AbstractRouterHandler<RouterHandlerFactory<CommonRouterHandler>>
```

Comment:
@Yuriy27 RouterHandlerFactory is implementation. We should not depend on implementation.